### PR TITLE
ZK-623: Relayer address endpoint

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
1. Add a new endpoint to the relayer service (exposing fee address in plain text)
2. CLI: remove configuration command and app state field. Instead, ask for the fee address before every withdrawal
3. TS: similar thing as CLI

### CODE REVIEW

File: crates/shielder-cli/src/app_state.rs [hunk 1]
```diff
@@ -36,6 +36,10 @@ impl RelayerRpcUrl {
         format!("{}/quote_fees", self.base_url)
```
Comment (guspiel) [inline crates/shielder-cli/src/app_state.rs:36]: Is this change supposed to be a part of the browser extension release? If yes, after merging we should publish the mpm using [instructions](https://www.notion.so/cardinalcryptography/Publishing-SDK-Relayer-and-Dev-Environment-134af09db1a980ab9bf2e04f8591a78a) and [workflow](https://github.com/Cardinal-Cryptography/zkOS-monorepo/actions/workflows/manual-publish-dev-env.yml), and then coordinate with Piotr Dżaluk to make sure the change is included in browser extension code.

**Review by guspiel — APPROVED** [review]
Looks reasonable to me. Tests passed so it seems to work (the typescript tests are run with an actual relayer). The thread that makes the "/fee_address" HTTP query is the same thread as the one that later runs "/relay", so we're not introducing any delays to a thread that is expected to be responsive.

File: ts/shielder-sdk-tests/web/shielder/testUtils.ts [hunk 2]
```diff
@@ -83,10 +83,15 @@ export class MockedContract implements IContract {
```
[the remark this answers is not in the document]
Comment (pmikolajczyk41) [reply ts/shielder-sdk-tests/web/shielder/testUtils.ts:83]: it is used in the implementation of `address()` function (as a memorized value)

**Review by DamianStraszak — APPROVED** [review]
Looks great!

**Pull request merged**